### PR TITLE
refactor: Use `@ant-design/fast-color` instead

### PR DESCRIPTION
**Pull request by zombieJ**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Replace ColorPicker internal `@ctrl/tiny-color` with `@ant-design/fast-color` to reduce bundle size.      |
| 🇨🇳 Chinese |    替换 ColorPicker 内部使用的 `@ctrl/tiny-color` 为 `@ant-design/fast-color` 以降低打包尺寸。      |

### CODE REVIEW

**Comment by afc163** [comment]
随便问一嘴，我们没有把 TinyColor 实例对象暴露给用户吧，如果暴露了底层换成别的 color 这就是 break change。

**Comment by afc163** [comment]
图片

反而还大了

**Comment by zombieJ** [comment]
嗯，很多地方都用了 tinyColor 要慢慢剥掉

**Review by yoyo837 — APPROVED** [review]

File: components/color-picker/__tests__/__snapshots__/demo.test.tsx.snap [hunk 1]
```diff
@@ -265,7 +265,7 @@ exports[`renders components/color-picker/demo/panel-render.tsx correctly 1`] = `
           >
             <div
               class="ant-color-picker-color-block-inner"
-              style="background:rgb(22, 119, 255)"
+              style="background:rgb(22,119,255)"
```
Comment (crazyair) [inline components/color-picker/__tests__/__snapshots__/demo.test.tsx.snap:268]: 这个规范要空格还是不要空格

Comment (afc163) [reply components/color-picker/__tests__/__snapshots__/demo.test.tsx.snap:268]: 这是 snap ，可以不用关系，jest 跑出来是啥就是啥。

Comment (zombieJ) [reply components/color-picker/__tests__/__snapshots__/demo.test.tsx.snap:268]: fastColor 里省空间 & 性能所以不加这个

**Comment by eladonline** [comment]
this change created a failiure in the compile 
image

**Comment by yoyo837** [comment]
> this change created a failiure im using CRA - react-scripts V4 I think it might be related to babel versions collision image

Fixed by https://github.com/ant-design/fast-color/pull/50, Please try remove lock file and `node_modules` then reinstall it.

**Comment by eladonline** [comment]
@yoyo837 

- antd@5.20.0
33 minutes ago remove node-modules and package-lock 
still same issue

**Comment by crazyair** [comment]
`@ant-design/fast-color` version ？

**Comment by eladonline** [comment]
> @ant-design/fast-color

└─┬ antd@5.20.0
  └─┬ @rc-component/color-picker@2.0.0
    └── @ant-design/fast-color@2.0.5

**Comment by zombieJ** [comment]
Updated `@ant-design/fast-color` compile target version. Pls reinstall again. But I think the best way is to upgrade compile framework version since it's already support by modern browsers.

**Comment by eladonline** [comment]
> Updated `@ant-design/fast-color` compile target version. Pls reinstall again. But I think the best way is to upgrade compile framework version since it's already support by modern browsers.

image

Im so sorry :)

**Comment by eladonline** [comment]
> Updated `@ant-design/fast-color` compile target version. Pls reinstall again. But I think the best way is to upgrade compile framework version since it's already support by modern browsers.

seems to work just fine now

**Pull request merged**